app/grandchallenge/products/templates/products/partials/product_list.html

```diff
@@ -31,16 +31,30 @@
         </div>
       </div>
 
+
       <div class="col-sm-3 bg-light pt-2">
         <div class="d-flex justify-content-between flex-column h-100">
           <div>
             <div class="d-flex justify-content-between">
-              <div class="h6 text-muted mr-auto pb-1">CE: </div>
-              <div class="h6 pr-2">{% if product.ce_class %}{{product.ce_class}}{% endif %}</div>
+                <a href="#" data-toggle="tooltip" data-placement="top" class="h6 text-muted mr-auto pb-1"
+                   title= {% include "products/partials/tooltip_CE.html" %}>
+                   CE:
+                </a>
+              <div class="h6 pr-2">
+                  {% if product.ce_class %}
```

Comment:
I think it's better to put the `if` statement around the `<div class="h6 pr-2">`, otherwise you'll add some extra whitespace for items with no CE class. Unless that's what you want of course.

Reply:
fixed